03:24:57 PM: DEBUG [logger_base.py:12] Mensaje a nivel debug
03:24:57 PM: INFO [logger_base.py:13] Mensaje a nivel info
03:24:57 PM: WARNING [logger_base.py:14] Mensaje a nivel de warning
03:24:57 PM: ERROR [logger_base.py:15] Mensaje a nivel de error
03:24:57 PM: CRITICAL [logger_base.py:16] Mensaje a nivel critico
03:25:02 PM: DEBUG [logger_base.py:12] Mensaje a nivel debug
03:25:02 PM: INFO [logger_base.py:13] Mensaje a nivel info
03:25:02 PM: WARNING [logger_base.py:14] Mensaje a nivel de warning
03:25:02 PM: ERROR [logger_base.py:15] Mensaje a nivel de error
03:25:02 PM: CRITICAL [logger_base.py:16] Mensaje a nivel critico
03:32:58 PM: DEBUG [conexion.py:24] Conexión exitosa: <connection object at 0x0000019921877230; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
03:32:58 PM: DEBUG [conexion.py:37] Se abrió correctamente el cursor: <cursor object at 0x000001992334B4C0; closed: 0>
03:32:58 PM: DEBUG [persona_dao.py:54] Objeto eliminado: 
            Id Persona: 20, Nombre: None,
            Apellido: None, Email: None
        
03:32:58 PM: DEBUG [persona_dao.py:72] Personas eliminadas: 0
03:32:58 PM: DEBUG [persona_dao.py:77] 
            Id Persona: 2, Nombre: Ivonne,
            Apellido: Gutierrez, Email: igutierrez@mail.com
        
03:32:58 PM: DEBUG [persona_dao.py:77] 
            Id Persona: 3, Nombre: Carlos,
            Apellido: Lara, Email: clara@mail.com
        
03:32:58 PM: DEBUG [persona_dao.py:77] 
            Id Persona: 4, Nombre: Marcos,
            Apellido: Cantu, Email: mcantu@mail.com
        
03:32:58 PM: DEBUG [persona_dao.py:77] 
            Id Persona: 5, Nombre: Angel,
            Apellido: Quintana, Email: aquintana@mail.com
        
03:32:58 PM: DEBUG [persona_dao.py:77] 
            Id Persona: 6, Nombre: Maria,
            Apellido: Gonzalez, Email: mgonzalez@mail.com
        
03:32:58 PM: DEBUG [persona_dao.py:77] 
            Id Persona: 7, Nombre: Alex,
            Apellido: Rojas, Email: arojas@mail.com
        
